sqlmesh/core/engine_adapter/base.py

```diff
@@ -407,6 +409,24 @@ def create_index(
         )
         self.execute(expression)
 
+    @staticmethod
+    def _extract_creatable_type_from_properties(
+        properties: t.Dict[str, exp.Expression],
+    ) -> t.Optional[exp.Expression]:
+        """Parses out the type from the properties dictionary and returns it if it exists. It does also remove the key.
```

Comment:
Let's make sure to the "Parses" bit here and in any other places too.

Reply:
I renamed this slightly to be closer to Python conventions.